nginx/nginx.conf

```
@@ -0,0 +1,62 @@
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    upstream relay {
+        server relay:3000;
+    }
+
+    upstream sentry {
+        server web:9000;
+    }
+
+    server {
+        listen 80;
+        # use the docker DNS server to resolve ips for relay and sentry containers
+        resolver 127.0.0.11 ipv6=off;
+
```

Let's also squeeze in the max upload size limit change @jan-auer mentioned somewhere around here:
```suggestion
        client_max_body_size 32M;
```